Taskfile.yml

```diff
@@ -0,0 +1,189 @@
+
+  backend-pip-lock:
+    #TODO: Add a task to lock the pip dependencies
+    desc: Lock the pip dependencies
+    dir: tdrs-backend
+    cmds:
+      - docker-compose -f docker-compose.yml exec web sh -c "pipenv lock"
+
+
+  clean:
+    desc: Remove all containers, networks, and volumes
+    cmds:
+      - docker-compose -f tdrs-backend/docker-compose.yml down -v
+      - docker-compose -f tdrs-frontend/docker-compose.yml down -v
+      - docker system prune -f -a
+  
+  clamav-up:
+    desc: Start clamav service
+    dir: tdrs-backend
+    cmds:
+      - docker-compose -f docker-compose.yml up -d clamav-rest
+
+  frontend-up:
+    desc: Start frontend web server
+    dir: tdrs-frontend
+    cmds:
+      - docker-compose -f docker-compose.yml up -d
+
+  frontend-down:
+    desc: Stop frontend web server
+    dir: tdrs-frontend
+    cmds:
+      - docker-compose -f docker-compose.yml down
+
+  frontend-restart:
+    desc: Restart frontend web server
+    dir: tdrs-frontend
+    cmds:
+      - docker-compose -f docker-compose.yml restart -d
+
+  frontend-av:
+    desc: Start frontend with optional clamav service
+    dir: tdrs-frontend
+    cmds:
+      - task: frontend-up
+      - task: clamav-up
+
+  
+  # need more work
+  frontend-init:
+    desc: Initialize the frontend project
+    dir: tdrs-frontend
+    cmds:
+      - docker-compose -f docker-compose.yml up -d --build
+      - docker-compose -f docker-compose.yml exec tdp-frontend sh -c "apk add nodejs npm"
+      - docker-compose -f docker-compose.yml exec tdp-frontend sh -c "npm install"
+      - docker-compose -f docker-compose.yml down
+
+  frontend-test:
+    desc: Run frontend tests
+    dir: tdrs-frontend
+    cmds:
+      - docker-compose -f docker-compose.local.yml up tdp-frontend-test -d
+      - docker-compose -f docker-compose.local.yml exec tdp-frontend-test sh -c "npm run test"
+
+  frontend-test-cov:
+    desc: Run frontend tests with coverage
+    dir: tdrs-frontend
+    cmds:
+      - docker-compose -f docker-compose.local.yml up tdp-frontend-test -d
+      - docker-compose -f docker-compose.local.yml exec tdp-frontend-test sh -c "npm run test:cov"
+
+  frontend-lint:
+    desc: Run eslint in the frontend container
+    dir: tdrs-frontend
+    cmds:
+      - docker-compose -f docker-compose.local.yml up -d tdp-frontend-test --quiet-pull
+      - docker-compose -f docker-compose.yml exec tdp-frontend-test sh -c "npm run lint"
+
+  frontend-logs:
+    desc: Show and follow frontend web server logs
+    dir: tdrs-frontend
+    cmds:
+      - docker-compose -f docker-compose.yml logs -f
+
+  frontend-bash:
+    desc: Open a shell in the frontend container
+    dir: tdrs-frontend
+    cmds:
+      - docker-compose -f docker-compose.yml exec tdp-frontend sh
+
+  frontend-pip-lock:
+    desc: Lock the pip dependencies
+    dir: tdrs-frontend
+    cmds:
+      - task: frontend-up
+      - docker-compose -f docker-compose.yml exec tdp-frontend sh -c "pipenv lock"
```

Comment:
pipenv shouldnt exist in this container right?

Reply:
oops, this is a typo, from older merge